### PR TITLE
feat(lang): Chinese translation update (v1.0.0-beta2 & beta3)

### PR DESCRIPTION
**Pull request by seognil**
Mainly two things
* fix little English version error
* update Chinese version text and config

### CODE REVIEW

File: content/v1.0.0-beta.2/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ consumers of your library:
...
 1. **BREAKING CHANGE:** a commit that has the text `BREAKING CHANGE:` at the beginning of its optional body or footer section introduces a breaking API change (correlating with [`MAJOR`](http://semver.org/#summary) in semantic versioning). A breaking change can be
   part of commits of any _type_. e.g., a `fix:`, `feat:` & `chore:` types would all be valid, in addition to any other _type_.
-1. Others: commit _types_ other than `fix:` and `feat:` are allowed, for example [commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional) (based on the [the Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines)) recommends `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others. We also recommend `improvement` for commits that improve a current implementation without adding a new feature or fixing a bug. Notice these types are not mandated by the conventional commits specification, and have no implicit effect in semantic versioning (unless they include a BREAKING CHANGE, which is NOT recommended).
```
Comment (damianopetrungaro) [inline content/v1.0.0-beta.2/index.md]: This changes should not be there 😄

File: content/v1.0.0-beta.2/index.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ It discourages moving fast in a disorganized way. It helps you be able to move f
 
 ### Might Conventional Commits lead developers to limit the type of commits they make because they'll be thinking in the types provided?
 
-Conventional Commits encourages us to make more of certain types of commits such as fixes. Other than that, the flexibility of Conventional Commits allows your team to come up with their own types and change those types over time.
```
Comment (damianopetrungaro) [inline content/v1.0.0-beta.2/index.md]: These changes should not be there 😄

File: content/v1.0.0-beta.3/index.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ consumers of your library:
 1. **feat:** a commit of the _type_ `feat` introduces a new feature to the codebase (this correlates with [`MINOR`](http://semver.org/#summary) in semantic versioning).
 1. **BREAKING CHANGE:** a commit that has the text `BREAKING CHANGE:` at the beginning of its optional body or footer section introduces a breaking API change (correlating with [`MAJOR`](http://semver.org/#summary) in semantic versioning).
 A BREAKING CHANGE can be part of commits of any _type_.
-1. Others: commit _types_ other than `fix:` and `feat:` are allowed, for example [commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional) (based on the [the Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines)) recommends `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others.
```
Comment (damianopetrungaro) [inline content/v1.0.0-beta.3/index.md]: These changes should not be there 😄

File: content/v1.0.0-beta.2/index.md (hunk 1 above)
Comment (seognil) [reply content/v1.0.0-beta.2/index.md]: `[commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional)`
is a dead link now, that repo has moved to a new place
I just updated the link.

File: content/v1.0.0-beta.3/index.md (hunk 3 above)
Comment (seognil) [reply content/v1.0.0-beta.3/index.md]: same thing, dead link

File: content/v1.0.0-beta.2/index.md (hunk 1 above)
Comment (damianopetrungaro) [reply content/v1.0.0-beta.2/index.md]: So let's fix it in another PR (or at least change the name of the PR)

File: content/v1.0.0-beta.2/index.md (hunk 2 above)
Comment (seognil) [reply content/v1.0.0-beta.2/index.md]: add backquote to the word `fixes`
(I guess it's supposed to be?)

File: content/v1.0.0-beta.2/index.md (hunk 1 above)
Comment (seognil) [reply content/v1.0.0-beta.2/index.md]: Oh yeah, you are right, thanks
(never thought that before... 😥)
I'll make a new commit to reverse (manually) these changes and a new PR soon

Comment (seognil) [reply content/v1.0.0-beta.2/index.md]: @damianopetrungaro 
I'm thinking should I rework my branch to only output fewer cleaner commits (is that ok for the github review system?)
or close this PR and make a brand new one
or is current commit history for this PR acceptable

**Comment by stevemao** [comment]
cc @doodlewind

**Comment by damianopetrungaro** [comment]
Ty @seognil for the huge effort and @doodlewind for the native review :D

**Pull request merged**